custom-metrics-autoscaling/direct-to-sd/custom-metrics-sd.yaml

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START all]
-apiVersion: apps/v1
+# [START gke_autoscale_stackdriver_manifest]
```

Comment:
For any YAML sample I would include the `kind` in the region tag name, you're already doing it with some of these. for instance include `deployment` here - not `manifest`